MachObfuscator/Mach/Mach+Erasing.swift

```diff
@@ -46,4 +54,19 @@ private extension Mach {
         }
         data.nullify(range: sect.range.intRange)
     }
+
+    mutating func eraseFilePaths(_ prefixes: [String], usingReplacement replacement: String) {
+        guard !prefixes.isEmpty else {
+            // nothing to do
+            return
+        }
+
+        data.replaceStrings(inRange: cstringSection!.range.intRange, withMapping: {
```

Comment:
`cstringSection` is rather unlikely to be nil in any application, so this will probably not be an issue, but I have to write that:
What about `guard let`ing the `cstringSection` instead of force unwrapping. Obfuscation should not fail, when there is no `cstringSection`, should it?

Reply:
Ok, I added `guard let`, but I also think that images without `cstrings` are very unlikely or even impossible (the complier also puts some texts in there).